string topicName
bool newValue
---
string setAnswer
